shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -144,6 +144,10 @@ public boolean isBulkUserEmailEmailSendingEnabled() {
                 .equals("true");
     }
 
+    public boolean isAuthCodeHandlerEnabled() {
```

Comment:
Do we want to feature flag this independently of the wider auth orch split? We have this further up:

```
public boolean isAuthOrchSplitEnabled() {
        return System.getenv().getOrDefault("SUPPORT_AUTH_ORCH_SPLIT", "false").equals("true");
    }
```

To me it feels more of a faff than it's worth to toggle all the lambdas separately, but if you disagree and/or already discussed with James, fair enough